backend/build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
     id 'org.springframework.boot' version '2.7.6'
     id 'org.liquibase.gradle' version '2.0.4'
-    id 'io.spring.dependency-management' version '1.0.15.RELEASE'
```

Comment:
I don't see a particular reason for us to move off the RELEASE version type to a specific version - any objections to just closing this PR?

CC @alismx @rin-skylight @mehansen @zdeveloper